src/custom_api_response.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# heritedcode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/heritedcode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+class CustomApiResponse:
+    """
+    this class represents response data structure for returning response
+    from functions.
+    """
+
+    def __init__(self, data):
+        for key in data:
+            self.key = data[key]
+
+    """
```

Comment:
A docstring should right after a function definition, also you may want to check the spelling: obj? claa? 
But here, the `__eq__` dunder method being a special well known method, there is really no need a docstring at all ... so you are better off not having one entirely